pages/_app.tsx

```diff
@@ -0,0 +1,40 @@
+import "../styles/globals.css";
+import type { AppProps } from "next/app";
+import {
+  LivepeerConfig,
+  createReactClient,
+  studioProvider,
+} from "@livepeer/react";
+import { Toaster } from "react-hot-toast";
+import {
+  ThirdwebProvider,
+  metamaskWallet,
+  coinbaseWallet,
+  walletConnectV1,
+  walletConnect,
+  safeWallet,
+  paperWallet,
+} from "@thirdweb-dev/react";
+
+const client = createReactClient({
+  provider: studioProvider({ apiKey: "9e8e8a41-24c3-42ec-9b7e-bc3739177068" }),
+});
+
+export default function App({ Component, pageProps }: AppProps) {
+  return (
+    <ThirdwebProvider
+      supportedWallets={[
+        metamaskWallet(),
+        coinbaseWallet(),
+        walletConnect(),
+        safeWallet(),
+      ]}
+      activeChain="ethereum"
```

Comment:
is the app only gonna be used in ethereum? why is there a state to switch chains on the other component?

Reply:
can you please share which state are you referring to?

Reply:
you're hardcoding ethereum, asking if that's intended